### DISCUSSION TITLE
Walk me through it like I'm your 90 year old (grand)parent

### DISCUSSION THREAD

**Discussion by quietfangirl** [lightspark/lightspark#802] [Q&A] [answered]
So I'm trying to install lightspark on the Firefox browser on a Windows 10 computer, both of which are the most up-to-date version. I cannot for the life of me find _anything_ about getting new plugins, installing plugins, or lightspark as an extension. Mozilla is being anything _but_ helpful, and all the other tips I've seen about installing plugins generally starts with something along the lines of "go to add/find/search plugins..." which I either **don't have** or **can't find.**
So if someone can explain to me how to get lightspark on Firefox from the very beginning, pretending I'm an elderly woman who has never seen a computer before and just wants to know why her flash games aren't showing up anymore, that would be lovely. I'm not technologically illiterate, and I think I'm a pretty smart person, but this stuff is so far out of my areas of expertise it's like trying to read something in the Cyrillic alphabet and I can't even figure out which Slavic language it is.

**Comment by huwdp** [comment, marked as answer]
Firefox and Chromd have basically removed Flash support and other plugin support such as Lightspark. I'd give Pale Moon here (https://www.palemoon.org) a go and see if it works.

**Reply by eiglow** [reply]
There is also a wiki page which explains this more https://github.com/lightspark/lightspark/wiki/Getting-Lightspark-up-and-running-in-Web-Browser

**Reply by quietfangirl** [reply]
Thank you! I was really worried about asking this since it's not your job to play tech support, I'll check out the wiki and Pale Moon and see if I can get my flash games up and running ^-^

**Reply by nofreeusernames** [reply]
So It work?

**Reply by huwdp** [reply]
@nofreeusernames it worked by the looks of things.
